corehq/apps/data_interfaces/models.py

```diff
@@ -141,8 +141,20 @@ class AutomaticUpdateRuleCriteria(models.Model):
     class Meta:
         app_label = "data_interfaces"
 
+    def _get_case_value(self, case, property_name):
+        if property_name.lower().startswith('parent/'):
```

Comment:
not sure if this is true, but wonder if there any app manager code we could use that abstracts this out somehow (thinking of the code that resolves case property references)